bin/drop_filter_results.py

```diff
@@ -148,14 +151,27 @@ def filter_fraser_result(
         df_results_family_aberrant_splicing, out_drop_gene_name
     )
     file_name_research = f"{case_id}{output_name_fragment_as}_research.tsv"
-    df_results_family_aberrant_splicing.to_csv(
+    # df_results_family_aberrant_splicing = camelize_dataframe_columns(df_results_family_aberrant_splicing)
+    camelize_dataframe_columns(df_results_family_aberrant_splicing).to_csv(
         file_name_research, sep="\t", index=False, header=True
     )
     filter_by_gene_panel(
         df_results_family_aberrant_splicing, gene_panel, file_name_research
     )
 
 
+def camelize_dataframe_columns(df):
+    def camelize(string):
+        if match("^[a-z]+([A-Z][a-z0-9]*)*$", string):
+            return string
+        parts = split("([^a-zA-Z0-9])", string)
+        camelized = "".join(part.capitalize() for part in parts if part.isalnum())
+        return camelized[0].lower() + camelized[1:]
```

Comment:
We could add a fallback to return the original string in the case where the regex doesn't match. 
Chatgpt provided me with this:
```python
 # Return camelized string or original if empty
return camelized[0].lower() + camelized[1:] if camelized else string
```

Reply:
Done